--- a/package/boot/uboot-tools/uboot-envtools/files/mediatek_filogic
+++ b/package/boot/uboot-tools/uboot-envtools/files/mediatek_filogic
@@ -140,9 +140,12 @@
 zyxel,ex5700-telenor)
 	ubootenv_add_uci_config "/dev/ubootenv" "0x0" "0x4000" "0x4000" "1"
 	;;
+cmcc,mr3000d-ciq-256m)
+	ubootenv_add_uci_config "/dev/mtd1" "0" "0x20000" "0x20000" 1
+	;;
 esac
 
 config_load ubootenv
 config_foreach ubootenv_add_app_config
 
 exit 0
